docs/graph-algorithms/prim-algorithm.mdx

```diff
@@ -0,0 +1,224 @@
+---
+title: Prim Algorithm
+slug: /graph-algorithms/prim-algorithm
+---
+
+## Introduction
+
+The Prim's algorithm is a greedy algorithm. This algorithm is used to find a minimum spanning tree for a weighted undirected graph. This means it finds a subset of the edges that forms a tree that includes every vertex, where the total weight of all the edges in the tree is minimized.
+
+## Explanation
+
+From a given undirected graph find the path such that it covers all the edges and has minimum total weight.
+
+### Quick Example :
+
+![Step1](https://cdn.programiz.com/sites/tutorial2program/files/pa_1.png)
+![Step2](https://cdn.programiz.com/sites/tutorial2program/files/pa_2.png)
+![Step3](https://cdn.programiz.com/sites/tutorial2program/files/pa_3.png)
+![Step4](https://cdn.programiz.com/sites/tutorial2program/files/pa_4.png)
+![Step5](https://cdn.programiz.com/sites/tutorial2program/files/pa_5.png)
+![Step6](https://cdn.programiz.com/sites/tutorial2program/files/pa_6.png)
```

Comment:
Don't use images that you have not made yourself. Especially not from sites that are built for the same purpose as `NeoAlgo-Docs` 🤦